azure-kusto-ingest/azure/kusto/ingest/_resource_manager.py

```diff
@@ -88,7 +93,9 @@ def _get_resource_by_name(self, table: KustoResultTable, resource_name: str):
         return [_ResourceUri.parse(row["StorageRoot"]) for row in table if row["ResourceTypeName"] == resource_name]
 
     def _get_ingest_client_resources_from_service(self):
```

Comment:
Do we want throttling checks only on get resources? why not all queries? what sparked this PR?

Reply:
Only for get resources and get token. These two operations are the most likely to get throttled.